### PR TITLE
fix tests

### PR DESCRIPTION
**Pull request by joshes**

### CODE REVIEW

**Comment by joshes** [comment]
hmm. seems that the "quit" test is hanging. works locally fine, will have to try and reproduce.

**Comment by joshes** [comment]
Couldn't see the forest for the trees. after re-looking at this, it's not good to expose server shutdown via the API anyway. So I've removed that functionality and the associated test. Coincidentally, that test was making it difficult to test. 

It seems that there are still issues with flake / pypy environments, but that appears due to environmental conditions, not code.

Should we just remove those envs? I'm not sure how to add them via Travis.

File: boatd/api.py [hunk 1]
```diff
@@ -51,14 +49,13 @@ def get(self):
...
     def post(self):
-        data = tornado.escape.json_decode(self.request.body)
-        response = {}
-        if data.get('quit'):
-            # FIXME: doesn't actually quit the server
-            self.running = False
-            response['quit'] = True
-
-        self.write(response)
+        try:
```
Comment (kragniz) [inline boatd/api.py]: I think we can just remove this whole function?

**Comment by kragniz** [comment]
I agree on the quit function, I've no idea why I added it.

pypy can be disabled by removing this line https://github.com/boatd/boatd/blob/master/.travis.yml#L6

**Comment by joshes** [comment]
alrighty, fixed, squashed and green!

**Pull request merged**